src/components/cesium_map/api/query.js

```diff
@@ -27,7 +27,7 @@ const geodist = (lat, long) => {
  * @returns
  */
 const setSolrQuery = ({ Q, searchFields, rows, field, lat, long }) => {
-  const fieldsParams = buildQuery(searchFields);
+  const fieldsParams = buildQuery(searchFields.concat([{field: "-relation_target", value: "*"}]));
```

Comment:
Since these APIs are outside `solr-faceted-search-react` framework, we need to exclude the relation child here.